iOSDevUK/Presentation/HomeView/HomeView.swift

```diff
@@ -22,6 +22,8 @@ struct HomeView: View {
         GridItem(.adaptive(minimum: 120))
     ]
 
+    @State private var shuffledSpeakers: [Speaker] = []
```

Comment:
shuffle them in the viewmodel.
Also, add shuffling on the AllSpeakersView as well.

Reply:
Thanks! Are you sure AllSpeakersView should be shuffled too?
My understanding was that HomeView is just a sneak peek into who the speakers are. So we don't expect users to use this section to actually find someone. Whereas AllSpeakersView is the place to go if you need a particular person. So if shuffled - it can be pretty hard to navigate there.